olive/cli/capture_onnx.py

```diff
@@ -0,0 +1,228 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+import tempfile
+from argparse import ArgumentParser
+from copy import deepcopy
+from pathlib import Path
+from typing import ClassVar, Dict
+
+from olive.cli.base import (
+    BaseOliveCLICommand,
+    add_remote_options,
+    get_model_name_or_path,
+    is_remote_run,
+    update_remote_option,
+)
+from olive.common.utils import IntEnumBase, set_tempdir
+
+logger = logging.getLogger(__name__)
+
+
+class ModelBuilderAccuracyLevel(IntEnumBase):
+    fp32 = 1
+    fp16 = 2
+    bf16 = 3
+    int8 = 4
+
+
+class CaptureOnnxGraphCommand(BaseOliveCLICommand):
+    allow_unknown_args: ClassVar[bool] = True
+
+    @staticmethod
+    def register_subcommand(parser: ArgumentParser):
+        sub_parser = parser.add_parser(
+            "capture-onnx-graph",
+            help=("Capture ONNX graph using PyTorch Exporter or Model Builder from the Huggingface model."),
```

Comment:
Huggingface only? This should work for any PyTorch model. Doesn't it??

Reply:
HF only. To work for any PT model, user need to provide 1) torch.nn network and 2) io_config among other things. We don't capture these info in CLI yet.